wireapp/wire-ios-mono
# chore: separate snapshot test target from test target - WPB-3109

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Unit tests and snapshot tests are a single target. This PR separates them into two separate targets for easier run.

### Solutions

1. Created another target & test plans for snapshot tests.
2. Moved snapshot tests into separate directory tree.

### Testing

#### How to Test

Pull the changes and verify test targets existence. Run both tests and snapshot tests for Wire-iOS.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
